tests/integration-tests/tests/common/utils.py

```diff
@@ -49,6 +49,8 @@
     "rhel8": {"name": "RHEL-8.8*_HVM-202309*", "owners": ["309956199498", "841258680906", "219670896067"]},
     # FIXME: unpin once Lustre client is available for Rocky 8.9
     "rocky8": {"name": "Rocky-8-EC2-Base-8.8*", "owners": ["792107900819"]},  # TODO add china and govcloud accounts
+    "rhel9": {"name": "RHEL-9.3*_HVM-*", "owners": ["309956199498", "841258680906", "219670896067"]},
```

Comment:
These accounts are the same used in Rhel8 (same between Rocky8 and Rocky9. What about avoiding duplication and define them in constants?

Reply:
Done